### ISSUE TITLE
Exception: has() and whereHas() do not support MorphTo relationships.

### ISSUE THREAD

**Issue by 4n70w4** [TitasGailius/nova-search-relations#12]
It will be very useful to have support for searching for polymorphic relations.

```php
    public static $searchRelations = [
        'detailable' => ['title', 'inn'],
    ];
```

```php
    public function detailable() {
        return $this->morphTo('detailable');
    }
```

```
    "message": "has() and whereHas() do not support MorphTo relationships.",
    "exception": "RuntimeException",
    "file": "/var/www/app/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Concerns/QueriesRelationships.php",
    "line": 35,
    "trace": [
        {
            "file": "/var/www/app/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Concerns/QueriesRelationships.php",
            "line": 158,
            "function": "has",
            "class": "Illuminate\\Database\\Eloquent\\Builder",
            "type": "->"
        },
        {
            "file": "/var/www/app/vendor/titasgailius/search-relations/src/SearchesRelations.php",
            "line": 73,
            "function": "orWhereHas",
            "class": "Illuminate\\Database\\Eloquent\\Builder",
            "type": "->"
        },
        {
            "file": "/var/www/app/vendor/titasgailius/search-relations/src/SearchesRelations.php",
            "line": 57,
            "function": "applyRelationSearch",
            "class": "App\\Nova\\Organization",
            "type": "::"
        },
        {
```

**Comment by TitasGailius** [comment]
This is a limitation of the Laravel framework, not this package.

**Comment by TitasGailius** [comment]
https://github.com/TitasGailius/nova-search-relations/issues/5

**Comment by 4n70w4** [comment]
@TitasGailius okay, https://medium.com/laravel-nova/nova-morph-to-search-relations-2296dbb66906 - helps for me.

But it would be nice to add an exception handler to your plugin, and show a more useful message, for example, to a workaround page. It will be convenient for users.

**Comment by conormx** [comment]
@TitasGailius just an FYI: https://github.com/laravel/framework/pull/28928 is included in Laravel 5.8.

**Comment by jkudish** [comment]
@TitasGailius can you add support for whereHasMorph now that it is supported, please :)
